realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -101,7 +106,7 @@ private SyncUser(ObjectServerUser user) {
      * A user is invalidated when he/she logs out or the user's access token expires.
      *
      * @return current {@link SyncUser} that has logged in and is still valid. {@code null} if no user is logged in or the user has
-     *         expired.
+     * expired.
```

Comment:
Does we indent lines only for parameter descriptions? If not, we should update https://github.com/realm/realm-java/blob/master/CONTRIBUTING.md